CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+
+## [2.0.2] - 2017-11-03
```

Comment:
We actually leave the actual version bumps and dates to maintainers per: https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change Also as this is a feaure and we follow semver this would be a minor not a patch version bump. In this case I will fix it for you prior to merge as you had quick turn around.